src/server/json_family.cc

```diff
@@ -1770,17 +1718,10 @@ void JsonFamily::StrLen(CmdArgList args, ConnectionContext* cntx) {
   string_view key = ArgS(args, 0);
   string_view path = ArgS(args, 1);
 
-  error_code ec;
-  JsonExpression expression = ParseJsonPath(path, &ec);
-
-  if (ec) {
-    VLOG(1) << "Invalid JSONPath syntax: " << ec.message();
-    cntx->SendError(kSyntaxErr);
-    return;
-  }
+  PARSE_PATH_ARG(path);
```

Comment:
This is really a villain 'macro' macro even with an implicit variable popping up 😵 We often use stuff like ParseOrReply()

Reply:
Fixed using gcc/clang extension which I think fits better here.

Reply:
😱